tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Services/ContactDetailServiceTests.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+using NHSD.BuyingCatalogue.Ordering.Api.Services;
+using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.AutoFixture;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Services
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    [SuppressMessage("ReSharper", "NUnit.MethodWithParametersAndTestAttribute", Justification = "False positive")]
+    internal static class ContactDetailServiceTests
```

Comment:
Should be `ContactDetailsServiceTests` to match the service name.

Reply:
Updated service name to `ContactDetailsServiceTests`